src/app/public/styles/themes/modern/_type.scss

```diff
@@ -1,16 +1,87 @@
 @import '../../variables-public';
 
 .sky-theme-modern {
-  .sky-deemphasized {
+  .sky-font-body-default {
+    font-size: $sky-theme-modern-font-body-default-size;
+    font-weight: $sky-theme-modern-font-body-default-weight;
+  }
+  
+  .sky-font-body-sm {
+    font-size: $sky-theme-modern-font-body-sm-size;
+    font-weight: $sky-theme-modern-font-body-sm-weight;
+  } 
+  
+  .sky-font-body-lg {
+    font-size: $sky-theme-modern-font-body-lg-size;
+    font-weight: $sky-theme-modern-font-body-lg-weight;
+  } 
+  
+  sky-font-field-label {
+    color: $sky-theme-modern-font-field-label-color;
+    font-size: $sky-theme-modern-font-field-label-size;
+    font-weight: $sky-theme-modern-font-field-label-weight;
+  }
+  
+  .sky-font-deemphasized {
     color: $sky-theme-modern-font-deemphasized-color;
     font-size: $sky-theme-modern-font-deemphasized-size;
     font-style: $sky-theme-modern-font-deemphasized-style;
   }
 
-  .sky-emphasized {
+  .sky-font-emphasized {
     font-weight: $sky-theme-modern-font-emphasized-weight;
   }
+  
+  .sky-font-heading-1 {
```

Comment:
If we want both `hx` elements and any element with the `.sky-font-heading-x` class to have the same styles, I'd prefer these both be defined in the same place for ease of updating styles in the future. I'd move them to `_theme.scss` where you removed the `.sky-hx` classes.